source/puppet/4.1/config_ssl_external_ca.markdown

```diff
@@ -201,230 +107,70 @@ Credential                        | File location
-
-~~~
-[agent]
-ssl_client_ca_auth = $certdir/issuer.pem
-~~~
-
-The value should point to somewhere in the `$certdir`.
-
-Put the external credentials into the correct filesystem locations. You can run the following commands to find the appropriate locations:
-
-Credential                        | File location
-----------------------------------|------------------------------------------------
-Agent SSL certificate             | `puppet agent --configprint hostcert`
-Agent SSL certificate private key | `puppet agent --configprint hostprivkey`
-Root CA certificate               | `puppet agent --configprint localcacert`
-Intermediate CA certificate       | `puppet agent --configprint ssl_client_ca_auth`
-
-## Option 3: Two Intermediate CAs Issued by One Root CA
-
-This configuration uses different CAs to issue Puppet master certificates and Puppet agent
-certificates. This makes them easily distinguishable, and prevents any agent certificate from being
-usable for a Puppet master.
-
-                   +------------------------+
-                   |                        |
-                   |  Root self-signed CA   |
-                   |                        |
-                   +------+----------+------+
-                          |          |
-               +----------+          +------------+
-               |                                  |
-               v                                  v
-      +-----------------+                +----------------+
-      |                 |                |                |
-      |    Master CA    |                |    Agent CA    |
-      |                 |                |                |
-      +--------+--------+                +--------+-------+
-               |                                  |
-               |                                  |
-               v                                  v
-      +-----------------+                +----------------+
-      |                 |                |                |
-      | Master SSL Cert |                | Agent SSL Cert |
-      |                 |                |                |
-      +-----------------+                +----------------+
-
-In this configuration, Puppet agents are configured to only authenticate peer certificates issued by the Master CA. Puppet masters are configured to only authenticate peer certificates issued by the Agent CA.
+## Option 2: Puppet master functioning as an intermediate CA
 
-> **Note:** If you're using this configuration, you can't use the ActiveRecord inventory service backend with multiple Puppet master servers. Use [PuppetDB][] for the inventory service instead.
+The puppet master can operate as an intermediate CA to an external Root CA.  The puppet master cannot be an intermediate to an intermediate.  In this mode the puppet master CA is left enabled and generation of agent certificates can remain automated, however there are some limitations:
```

Comment:
Tiny little nit - we're inconsistent in when we capitalize the words "intermediate" and "root" when referring to CAs.